File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRS/comm/DanaRS_Packet_History_Daily.java
package info.nightscout.androidaps.plugins.PumpDanaRS.comm;

import java.util.Date;

import com.cozmo.danar.util.BleCommandUtil;

public class DanaRS_Packet_History_Daily extends DanaRS_Packet_History_ {

    public DanaRS_Packet_History_Daily() {
        super();
        opCode = BleCommandUtil.DANAR_PACKET__OPCODE_REVIEW__DAILY;
    }

    public DanaRS_Packet_History_Daily(Date from) {
        super(from);
        opCode = BleCommandUtil.DANAR_PACKET__OPCODE_REVIEW__DAILY;
    }

    @Override
    public String getFriendlyName() {
        return "REVIEW__DAILY";
    }
}

File: pump/danars/src/main/kotlin/app/aaps/pump/danars/comm/DanaRSPacketHistoryDaily.kt
package app.aaps.pump.danars.comm

import app.aaps.core.interfaces.logging.LTag
import dagger.android.HasAndroidInjector
import info.nightscout.androidaps.danars.encryption.BleEncryption

class DanaRSPacketHistoryDaily(
    injector: HasAndroidInjector,
    from: Long = 0
) : DanaRSPacketHistory(injector, from) {

    init {
        opCode = BleEncryption.DANAR_PACKET__OPCODE_REVIEW__DAILY
        aapsLogger.debug(LTag.PUMPCOMM, "New message")
    }

    override val friendlyName: String = "REVIEW__DAILY"
}